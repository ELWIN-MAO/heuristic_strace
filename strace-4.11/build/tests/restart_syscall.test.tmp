
mym_waitpid=103671

mym_waitpid=103671

mym_waitpid=103671

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=103671

mym_waitpid=103671

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=103671

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103671

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103671

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103671

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103671

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=103671

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=103671

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=103671

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=103671

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=103671

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=103671

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=103671

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103671

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=103671

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=103671

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=103671

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=103671

syscall_enter_mym_scno=13  sys_name=rt_sigaction

mym_waitpid=103671

syscall_exit_mym_scno=13  sys_name=rt_sigaction

mym_waitpid=103671

syscall_enter_mym_scno=14  sys_name=rt_sigprocmask

mym_waitpid=103671

syscall_exit_mym_scno=14  sys_name=rt_sigprocmask

mym_waitpid=103671

syscall_enter_mym_scno=38  sys_name=setitimer

mym_waitpid=103671

syscall_exit_mym_scno=38  sys_name=setitimer

mym_waitpid=103671

syscall_enter_mym_scno=35  sys_name=nanosleep
nanosleep({0, 222222222}, 
mym_waitpid=103671

syscall_exit_mym_scno=35  sys_name=nanosleep
{0, 110978029}
mym_scall_rval=0xffffffffffffffff
) = ? ERESTART_RESTARTBLOCK (Interrupted by signal)

mym_waitpid=103671
--- SIGALRM {si_signo=SIGALRM, si_code=SI_KERNEL} ---

mym_waitpid=103671

syscall_enter_mym_scno=219  sys_name=restart_syscall
restart_syscall(<... resuming interrupted nanosleep ...>
mym_waitpid=103671

syscall_exit_mym_scno=219  sys_name=restart_syscall

mym_scall_rval=0
) = 0

mym_waitpid=103671

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=103671

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=103671

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=103671

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=103671

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=103671
+++ exited with 0 +++
