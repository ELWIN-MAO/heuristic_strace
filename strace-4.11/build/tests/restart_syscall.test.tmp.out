nanosleep\(\{0, 222222222\}, \{0, 110978029\}\) = \? ERESTART_RESTARTBLOCK \(Interrupted by signal\)
--- SIGALRM \{si_signo=SIGALRM, si_code=SI_KERNEL\} ---
(nanosleep\((\{0, 222222222\}), 0x7ffee7217630|restart_syscall\(<\.\.\. resuming interrupted nanosleep \.\.\.>)\) = 0
\+\+\+ exited with 0 \+\+\+
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
